packages/beacon-node/package.json

```diff
@@ -76,8 +76,8 @@
     "lint": "eslint --color --ext .ts src/ test/",
     "lint:fix": "yarn run lint --fix",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:unit:minimal": "vitest --run --segfaultRetry 3 --dir test/unit/",
-    "test:unit:mainnet": "LODESTAR_PRESET=mainnet vitest --run --dir test/unit-mainnet",
+    "test:unit:minimal": "LODESTAR_PRESET=minimal vitest --run --segfaultRetry 3 --dir test/unit/",
+    "test:unit:mainnet": "LODESTAR_PRESET=mainnet vitest --run --segfaultRetry 3 --dir test/unit-mainnet",
```

Comment:
I noticed this `segfaultRetry` was added.  what does that do?

Reply:
It was earlier added in `minimal` preset and missing in `mainnet`. It just check if tests fails with a segfault error, it tries to rerun it.